db/tests/dependents/conftest.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import MetaData, Table, Column, Integer, PrimaryKeyConstraint, ForeignKeyConstraint
+import pytest
+
+from db.tables.operations.select import get_oid_from_table
+
+
+@pytest.fixture
+def post_comment_dependent_tables(engine_with_schema):
```

Comment:
It would be better to have fixtures based on the [Library use case](https://github.com/centerofci/mathesar-data-playground/tree/master/library_simulation/simulation_runs/20220801_00/sql_files) so that we can re-use it in other test cases too.